tests/Fixtures/TestBundle/ApiResource/Issue6264/BackedEnumStringTrait.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Tests\Fixtures\TestBundle\ApiResource\Issue6264;
+
+use ApiPlatform\Metadata\Operation;
+use Symfony\Component\Serializer\Attribute\Groups;
+
+trait BackedEnumStringTrait
```

Comment:
For further readers, this was duplicated from the BackedEnumTrait as graphql "guesses" the type from an union:

https://github.com/api-platform/core/blob/ef4812a688d4cd20877f5bad52d036e80cd43581/src/GraphQl/Type/FieldsBuilder.php#L232-L239

I'm not a huge fan, ideally we should provide a fix at some point there, even though when representing resources one should avoid **scalar** union types in my opinion.